noir_stdlib/src/merkle.nr

```diff
@@ -14,8 +14,8 @@ pub fn compute_merkle_root<N>(leaf: Field, index: Field, hash_path: [Field; N])
          } else {
             (current, hash_path[i])
          };
-
-      current = crate::hash::pedersen([hash_left, hash_right])[0];
+    // TODO(Kev): This should be changed to use pedersen_hash
+      current = crate::hash::pedersen_commitment([hash_left, hash_right])[0];
```

Comment:
Should have an issue open for this before merging -- this should use pedersen_hash because pedersen_commitment cannot be used as a hash function.

Reply:
Issue here: https://github.com/noir-lang/noir/issues/3343

Reply:
This is not a blocker for this PR because it was already there before this PR and is orthogonal -- changing to pedersen_commitment just made it more explicit